frontend/packages/dev-console/src/components/import/__tests__/DeployImage.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import { Provider } from 'react-redux';
+import { Radio } from '@patternfly/react-core';
+import store from '@console/internal/redux';
+import { PageHeading, ButtonBar } from '@console/internal/components/utils/';
+import NamespacedPage from '../../NamespacedPage';
+import DeployImagePage from '../DeployImagePage';
+import DeployImage from '../DeployImage';
+import ImageSearchSection from '../image-search/ImageSearchSection';
+import AppSection from '../app/AppSection';
+import AdvancedSection from '../advanced/AdvancedSection';
+import ResourceSection from '../section/ResourceSection';
+
+describe('DeployImage Page Test', () => {
+  type DeployImagePageProps = React.ComponentProps<typeof DeployImagePage>;
+  let deployImagePageProps: DeployImagePageProps;
+  let deployImagePageWrapper: ReactWrapper;
+  beforeAll(() => {
+    deployImagePageProps = {
+      history: null,
+      location: {
+        pathname: 'deploy-image/ns/openshift?preselected-ns=openshift',
+        search: 'deploy-image/ns/openshift?preselected-ns=openshift',
+        state: null,
+        hash: null,
+      },
+      match: {
+        isExact: true,
+        path: 'deploy-image/ns/openshift?preselected-ns=openshift',
+        url: 'deploy-image/ns/openshift?preselected-ns=openshift',
+        params: {
+          ns: 'my-project',
+        },
```

Comment:
nit: we can use same namespace in params and url/path

Reply:
updated